tests/test_main.py

```diff
@@ -62,14 +62,17 @@ def tearDown(self):
 
     def test_with_sam_file(self):
         sam_file = tempfile.NamedTemporaryFile(suffix=".sam", mode="w")
-        sam_file.write(b5_sam)
+        sam_file.write(b5_sam.encode())
         sam_file.seek(0)
         self.args.extend(["--sam-file", sam_file.name])
 
         # Set executable for BWA to "false"
         # This program always returns non-zero exit status
-        self.args.extend(["--method", "bwa"])
-        self.args.extend(["--bwa_fp", "false"])
+        config_file = tempfile.NamedTemporaryFile(suffix=".json")
```

Comment:
Config files are probably more trouble than they're worth.  Let's not remove the config files now, but just regard this as a sign that things will be much easier once we do.